setTRIMDAC_and_Mask.py

```diff
@@ -11,9 +11,10 @@
                   help="GTX on the GLIB", metavar="gtx")
 parser.add_option("-f", "--file", type="string", dest="trimfilelist",
                   help="File containing paths to MASK_TrimDACs", metavar="trimfilelist", default="TrimDACfiles.txt")
-
-parser.add_option("-d", "--debug", action="store_true", dest="debug",
-                  help="print extra debugging information", metavar="debug")
+parser.add_option("-t", "--thresh", action="store_true", dest= "do_thresh",
+                  help="Do a threshold scan before/after setting trim", metavar="do_thresh")
+parser.add_option("-d", "--debug", action="store_true", dest= "debug",
+                  help="Debugging mode", metavar="debug")
 
```

Comment:
Provide an option to use/not the ultra scan module, and then update the code accordingly